src/main/java/org/shanerx/tradeshop/Utils.java

```diff
@@ -735,13 +735,13 @@ public Block findShopChest(Block sign) {
     public void changeInvName(BlockState state, String name, List<OfflinePlayer> owners, List<OfflinePlayer> members) {
         StringBuilder sb = new StringBuilder();
         if (name.equalsIgnoreCase("") || name == null) {
-            name = state.getType().name().toString();
+            name = "";
         }
         sb.append(name + " <");
         owners.forEach(o -> sb.append("o:" + o.getName() + ";"));
         members.forEach(m -> sb.append("m:" + m.getName() + ";"));
         sb.append(">");
-        setName((InventoryHolder) state, sb.toString());
+        setName((InventoryHolder) state, sb.toString().replace("_", " "));
```

Comment:
I can’t follow... what is happening here? This would change usernames and could cause bugs...

Reply:
It was originally intended for material names, however thats not needed anymore so itll be removed as well